### ISSUE TITLE
URL/file source

### ISSUE THREAD

**Issue by davedoesdev** [davedoesdev/streamana#4]
Add option to broadcast from an audio / video URL or file.
Maybe list as an alternative input device?

**Comment by fuqet90** [comment]
Can it be possible to stream from video URL?

**Comment by davedoesdev** [comment]
Yes, it should be possible to stream from a video, just don't have time to update the example.
